### DISCUSSION TITLE
command node ace make:migration does not exist

### DISCUSSION THREAD

**Discussion by mortezabonak** [adonisjs/core#2480] [Help] [answered]
I want to make migration but its fails
`[ error ]  "make:migration" command not found`
node version: 14.15.3
npm version: 6.14.9
Framework version: 5.1.4        
Assembler version: 3.1.1
adonisjs/core: "^5.1.4"

I also entered command `node ace make:manifest` but the problem could not be solved.

**Comment by RomainLanz** [comment, marked as answer]
Hey @mortezabonak! 👋 

First, notice that you are using a not-released version of `@adonisjs/core`, please, stick to the version installed by the CLI.

Secondly, you need to install Lucid [as explain in our documentation](https://preview.adonisjs.com/guides/database/setup) to have this command available.

**Reply by mortezabonak** [reply]
thanks @RomainLanz 

its working now
